From 5711ccfab40e304ced7f5be39a4083e3fcecff91 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 24 May 2023 22:22:41 -0700
Subject: [PATCH] configure: Keep first line of compiler version string

Full output of cc -v may contain additional information which could
contain build path information, which is unnessasary

Upstream-Status: Submitted [https://gitlab.gnome.org/GNOME/gimp/-/merge_requests/926]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c25415d..68707a0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -599,7 +599,7 @@ fi
 for CC_VERSION_OPT in $CC_VERSION_OPTS; do
   # We run $CC, and escape and format its output, in a single step,
   # since some shells expand escape sequences in "echo" arguments.
-  CC_VERSION="`$CC $CC_VERSION_OPT 2>&1 | sed -e 's/\\\\/\\\\\\\\\\\\\\\\/g;s/^/\\\\t/;s/$/\\\\n/' | tr -d '\n'`"
+  CC_VERSION="`$CC $CC_VERSION_OPT 2>&1 | head -1 | sed -e 's/\\\\/\\\\\\\\\\\\\\\\/g;s/^/\\\\t/;s/$/\\\\n/' | tr -d '\n'`"
   if test $? -eq 0; then
     break
   fi
-- 
2.40.1

